packages/flutter/test/material/ink_paint_test.dart

```diff
@@ -8,6 +8,7 @@ import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 import '../rendering/mock_canvas.dart';
```

Comment:
`ink_paint_test`

Only substantive change here is on line 58: adding a splashFactory to a test that shouldn't need duplication for each splashFactory, but currently depends on the default value on the theme